pfSense-pkg-API/files/etc/inc/api.inc

```diff
@@ -411,6 +411,47 @@ function get_pfsense_version() {
     return $ver_data;
 }
 
+<<<<<<< Updated upstream
```

Comment:
Seems git has added some lines that are throwing errors in my tests. Lines 414, 415, and 454. I'll merge this if you'll resubmit with these lines removed.